src/tests/end-to-end/common/scan-for-accessibility-issues.ts

```diff
@@ -18,6 +19,14 @@ export interface PrintableResult {
 }
 
 export async function scanForAccessibilityIssues(page: Page, selector: string): Promise<PrintableResult[]> {
+    const axeIsUndefined = await page.evaluate(() => {
+        return (window as any).axe === undefined;
+    });
+
+    if (axeIsUndefined) {
+        await page.injectScriptFile(path.join(__dirname, '../../../../node_modules/axe-core/axe.min.js'));
```

Comment:
Not sure if the hardcoded path can be improved or not (like in... not have the `../../../../` in front of it) but also not too concerned neither. I guess it's up to you if you want to improve this or not

Reply:
Did some research and couldn't find a way to do this for now